lib/routers/attachment.js

```diff
@@ -0,0 +1,77 @@
+const { Router } = require('express');
+const Busboy = require('busboy');
+const crypto = require('crypto');
+const { v4: uuid } = require('uuid');
+const sharp = require('sharp');
+
+const { S3 } = require('@asl/service/clients');
+const { NotFoundError } = require('@asl/service/errors');
+
+module.exports = settings => {
+
+  const router = new Router();
+  const s3 = S3(settings);
+  const { Attachment } = settings.models;
+
+  router.post('/', async (req, res, next) => {
+    const busboy = Busboy({ headers: req.headers, filesLimit: 1 });
+    const id = uuid();
+    const token = crypto.randomBytes(64).toString('hex');
```

Comment:
Is the a reason for generating a random token as well as a UUID? Wouldn't the UUID effectively be random anyway?